templates/default/fulldoc/html/frames.erb

```diff
@@ -7,7 +7,9 @@
 <script type="text/javascript">
   var match = decodeURIComponent(window.location.hash).match(/^#!(.+)/);
   var name = match ? match[1] : '<%= url_for_main %>';
-  name = name.replace(/^((\w*):)?[\/\\]*/gm, '').trim();
+  if (name.match(/^((?:(?:[\w]*):)?[\/\\]*)/gm)[0]?.length > 0) {
```

Comment:
At this point wouldn't something like `new URL(name, window.location.href) === window.location.origin` be a lot simpler?

Reply:
I think so, would you like me to open a new pr?

Reply:
I think you can just accept the suggestion below (if you've tested this and it works)